tensorflow_addons/image/BUILD

```diff
@@ -36,169 +36,10 @@ filegroup(
 )
 
 py_test(
-    name = "color_ops_test",
+    name = "image_test",
     size = "small",
```

Comment:
So placing all tests under a single "size" means that we'll restrict tests to < 1min of test time. Do we think that is sufficient for https://github.com/tensorflow/addons/issues/1316 or would we like tighter bounds?